DFC.App.JobProfileOverview.IntegrationTests/Data/DataSeeding.cs

```diff
@@ -0,0 +1,107 @@
+﻿using DFC.App.JobProfileOverview.Data.Models;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Formatting;
+using System.Threading.Tasks;
+
+namespace DFC.App.JobProfileOverview.IntegrationTests.Data
+{
+    public class DataSeeding
+    {
+        private const string Segment = "segment";
+
+        public DataSeeding()
+        {
+            Article1Id = Guid.NewGuid();
+            Article2Id = Guid.NewGuid();
+            Article3Id = Guid.NewGuid();
+
+            Article1Name = "article1";
```

Comment:
Change to ArticleId[n].ToString() instead of hard coded name to make then unique